package.json

```diff
@@ -20,17 +20,21 @@
   "homepage": "https://github.com/scality/S3#readme",
   "dependencies": {
     "aws-sdk": "2.28.0",
-    "arsenal": "scality/Arsenal",
+    "arsenal": "scality/Arsenal#forward/orbit",
```

Comment:
in a DROPME commit ?

Reply:
missed that during rebase conflicts, we don't need the branch anymore

Reply:
pushed with the commit removed

Reply:
are you sure ? did you merge arsenal#forward/orbit in master already ? because there was differences (e.g. search disabled in master, enabled in fwd orbit)

Reply:
https://github.com/scality/Arsenal/blob/795b145594ea136eb24df05644bbf9a586df7a4f/lib/storage/metadata/mongoclient/MongoClientInterface.js#L751

vs

https://github.com/scality/Arsenal/blob/f3068144c76cd6fdb0e27c2881289ffbb27925da/lib/storage/metadata/mongoclient/MongoClientInterface.js#L736